polling_stations/apps/api/address.py

```diff
@@ -128,15 +129,18 @@ def get_ee_wrapper(self, address, query_params):
         if getattr(settings, "USE_LOCAL_PARQUET_ELECTIONS", False):
             helper = LocalParquetElectionsHelper()
             resp = helper.get_response(Postcode(address.postcode), address.uprn)
+            if not resp["request_success"]:
+                raise APIException("failed to get list of ballots")
```

Comment:
Inside WDIV (where we only care about polling stations), we have some handling for the `request_success=False` case. We can make some assumptions.
The API can't communicate this though, so instead of saying `ballots: []` we now throw an error, but only in the API. The devs.DC will handle these errors by wrapping them as a `UpstreamApiError` to forward on.
https://github.com/DemocracyClub/aggregator-api/blob/e016298ea2fa0f6186ed00fcc9c2a5482be2ffc1/api/endpoints/v1/voting_information/elections_api_client.py#L46-L47